systemtest/src/test/java/io/strimzi/systemtest/MirrorMakerST.java

```diff
@@ -352,6 +355,109 @@ void testWhiteList() throws Exception {
                 receiveMessages(messagesCount, kafkaClusterTargetName, false, topicNotInWhitelist, null, kafkaClientsPodName), is(0));
     }
 
+    @Test
+    void testCustomAndUpdatedValues() {
+        testMethodResources().kafkaEphemeral(CLUSTER_NAME, 1, 1)
+                .editSpec()
+                .withNewEntityOperator()
```

Comment:
just add tab two `.wthNewEntityOperator()` and `.endEntityOperator`

Reply:
I will add as part of review-commit :)